Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2915,7 +2913,7 @@ extension BrowserViewController: IntroViewControllerDelegate {
         }
     }
 
-    func presentSignInViewController(_ fxaOptions: FxALaunchParams) {
+    func presentSignInViewController(_ fxaOptions: FxALaunchParams? = nil) {
```

Comment:
There once call that uses these [params](https://github.com/mozilla-mobile/firefox-ios/pull/2691/files#diff-19fd1a6fe4b2f78eb457726f366733e0R324) still.